main/main.cpp

```diff
@@ -3211,6 +3212,9 @@ int Main::start() {
 #ifdef TOOLS_ENABLED
 		} else if (args[i] == "--no-docbase") {
 			gen_flags.set_flag(DocTools::GENERATE_FLAG_SKIP_BASIC_TYPES);
+		} else if (args[i] == "--gdextension-docs") {
```

Comment:
Interestingly Git doesn't raise a conflict here, but after #90705 this needs to be:
```suggestion
		} else if (E->get() == "--gdextension-docs") {
```